memor/session.py

```diff
@@ -103,69 +89,54 @@ def __add__(self, other_object):
             return Session(messages=new_messages)
         raise TypeError(UNSUPPORTED_OPERAND_ERROR_MESSAGE.format("+", "Session", type(other_object).__name__))
 
-    def __radd__(self, other_object):
+    def __radd__(self, other_object: Any) -> Any:
         """
         Reverse addition method.
 
         :param other_object: other object
-        :type other_object: any
-        :return: new Session
         """
         if isinstance(other_object, (Response, Prompt)):
             new_messages = [other_object] + self._messages
             return Session(title=self.title, messages=new_messages)
         raise TypeError(UNSUPPORTED_OPERAND_ERROR_MESSAGE.format("+", "Session", type(other_object).__name__))
 
-    def __contains__(self, message):
+    def __contains__(self, message: Union[Prompt, Response]) -> bool:
         """
         Check if the Session contains the given message.
 
         :param message: message
-        :type message: Response/Prompt
-        :return: bool
         """
         return message in self._messages
 
-    def __getitem__(self, index):
+    def __getitem__(self, index: int) -> Union[Prompt, Response]:
         """
         Return the Session message(s).
 
         :param index: index
-        :type index: int
-        :return: message(s) as Response/Prompt
         """
         return self._messages[index]
 
-    def __copy__(self):
-        """
-        Return a copy of the Session object.
-
-        :return: a copy of Session object
-        """
+    def __copy__(self) -> Any:
```

Comment:
Session

Reply:
Since it's in the same class it can not be used.